CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+### 2.26.0
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.9.4](https://fluentbit.io/announcements/v1.9.4/)
+* Amazon CloudWatch Logs for Fluent Bit 1.8.0
+* Amazon Kinesis Streams for Fluent Bit 1.9.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.25.1` this release adds:
+* Feature - Add `auto_create_stream ` option [cloudwatch:257](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/pull/257)
+* Enhancement - Add debug logs to check batch sizes [fluentbit:5428](https://github.com/fluent/fluent-bit/pull/5428)
+* Enhancement - Set 1 worker as default for `cloudwatch_logs` plugin [fluentbit:5417](https://github.com/fluent/fluent-bit/pull/5417)
+* Enhancement - Add `kube_token_ttl` option to kubernetes filter to support refreshing the service account token used to talk to the API server. Prior to this change Fluent Bit would only read the token on startup. [fluentbit:5332](https://github.com/fluent/fluent-bit/issues/5332)
```

Comment:
let's edit this comment making it clear that this was already cherry picked into our release but now is in upstream

Reply:
yeah I think we already had same situation before but I checked our previous changelog and we just did like this